app/views/appearances/edit.html.erb

```diff
@@ -0,0 +1,45 @@
+<% content_for :page_header do %>
+  <h1><span class="fa fa-paint-brush"></span> <%= t(:'sufia.admin.sidebar.appearance') %></h1>
+<% end %>
+
+<div class="row">
+  <div class="col-md-12">
+    <div class="panel panel-default">
+      <%= form_for(@site, url: site_appearances_path) do |f| %>
+        <div class="panel-heading">
+          <h3 class="panel-title">Site Appearance</h3>
```

Comment:
I don't think we've been consistent, but ought this use the i18n string `sufia.admin.sidebar.appearance`?